src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Unit.Tests/JsonSourceGeneratorTests.cs

```diff
@@ -768,5 +768,43 @@ internal partial class JsonContext : JsonSerializerContext
             Compilation compilation = CompilationHelper.CreateCompilation(source);
             CompilationHelper.RunJsonSourceGenerator(compilation);
         }
+
+
+// Enable once test harness is updated to be able to reference C# 12.
+#if false
```

Comment:
Skip attribute aren't used for things like this? Since at least those the test systems will know there's a test, whereas this pretty much requires somebody to remember.